### PR TITLE
Swap/gas button/tx confirmation screen improvements

### PR DESCRIPTION
**Pull request by christianbaroni**

### CODE REVIEW

File: src/components/gas/GasSpeedButton.js [hunk 1]
```diff
@@ -254,20 +257,18 @@ const GasSpeedButton = ({ onCustomGasBlur, onCustomGasFocus, type }) => {
...
-        message:
-          (tooLow
-            ? 'The gas price you set is too low and your transaction might get stuck or dropped.'
-            : 'We noticed the gas price you set is quite high.') +
-          '\n\nDo you want to continue anyway?',
-        title: 'Heads up!',
+        message: tooLow
+          ? 'If you don’t set a high enough gas price, your transaction '
+          : 'We noticed the gas price you set is quite high. Would you like to proceed anyway?',
+        title: 'Low gas price–transaction might get stuck!',
```
Comment (brunobar79) [inline src/components/gas/GasSpeedButton.js:271]: You need a different title for when is too high.

File: src/components/gas/GasSpeedButton.js [hunk 2]
```diff
@@ -254,20 +257,18 @@ const GasSpeedButton = ({ onCustomGasBlur, onCustomGasFocus, type }) => {
...
           },
         ],
-        message:
-          (tooLow
-            ? 'The gas price you set is too low and your transaction might get stuck or dropped.'
-            : 'We noticed the gas price you set is quite high.') +
-          '\n\nDo you want to continue anyway?',
-        title: 'Heads up!',
+        message: tooLow
+          ? 'If you don’t set a high enough gas price, your transaction '
```
Comment (brunobar79) [inline src/components/gas/GasSpeedButton.js:269]: This phrase is incomplete

**Review by brunobar79 — APPROVED** [review]
Looks great. Just need to fix the error msg strings.

File: src/components/gas/GasSpeedButton.js (hunk 2 above)
Comment (christianbaroni) [reply src/components/gas/GasSpeedButton.js:269]: forgot to finish these error messages, good catch

will fix

**Pull request merged**